src/modules/bulk-minter/components/Loading.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FunctionComponent } from "react";
+
+interface LoadingProps {
```

Comment:
Just a comment for other reviewers: I think we should save tailwind-ifying everything for a future PR where we overhaul this repo